docs/source/user-manual/navigation/katalogus.rst

```diff
@@ -0,0 +1,24 @@
+KAT-alogus
+==========
+
+The KAT-alogus (KAT catalog) is the place where you can see which plugins are available, enabled and/or disabled.
+Plugins can be common security scanning tools, like nmap (checks which ports are open), or specific tools that check for a CVE vulnerability.
+The KAT-alogus also contains all the Normalizers, which parse the data from the tools.
+Each plugin (Boefje and Normalizer) contains more information on how it works and what is required, including the Objects it can search for,
```

Comment:
```suggestion
Each plugin (Boefje and Normalizer) contains more information on how it works and what is required, including the objects it can search for,
```